src/main/java/nl/knaw/dans/prestaging/cli/LoadFromDataverseCommand.java

```diff
@@ -41,7 +42,7 @@
 import java.util.stream.Stream;
 import java.util.stream.StreamSupport;
 
-public class LoadFromDataverseCommand extends EnvironmentCommand<DdManagePrestagingConfiguration> {
+public class LoadFromDataverseCommand extends DefaultConfigEnvironmentCommand<DdManagePrestagingConfiguration> {
```

Comment:
Tried in https://github.com/DANS-KNAW-jp/dd-verify-file-migration/commit/4b7379f51861b71ffc9b4982db75726732896bf3

It gives the following helps:

    load-from-vault [-h] (-u UUIDS | -U UUID | -s STORE) file
    load-from-fedora [-h] [file] csv [csv ...]
    load-from-dataverse [-h] [-d DOI | --csv CSV] [file]

I don't yet understand why the help is still mandatory in one case, might be the mutually exclusive group doesn't inherit this environment variable. Still have to try with the environment variable.

Reply:
Not so strange, in `LoadFromVaultCommand` you have this code, instead of calling `super.configure`:
```java
        // mandatory variant of: super.addFileArgument(subparser);
        subparser.addArgument("file")
            .type(File.class)
            .required(true)
            .help("application configuration file");
```

Reply:
Thanks, thought I double checked but missed this third.